modules/using-aws-cli-create-iam-role-alb-operator.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * networking/installing-albo-sts-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="using-aws-cli-create-iam-role-alb-operator_{context}"]
+= Using the AWS CLI to create an IAM role for the Operator
+
+.Prerequisites
+
+* You must have access to the `aws` command line interface.
+
+.Procedure
+
+. Generate a trust policy file using your identity provider by running the following command:
++
+[source,terminal]
+----
+$ cat <<EOF > albo-operator-trust-policy.json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "Federated": "arn:aws:iam::777777777777:oidc-provider/d3gt1gce2zmg3d.cloudfront.net/25bslnj8bdll0fqhb9h5u0rmbpb3esgm" <1>
+            },
+            "Action": "sts:AssumeRoleWithWebIdentity",
+            "Condition": {
+                "StringEquals": {
+                    "d3gt1gce2zmg3d.cloudfront.net/25bslnj8bdll0fqhb9h5u0rmbpb3esgm:sub": "system:serviceaccount:aws-load-balancer-operator:aws-load-balancer-operator-controller-manager" <2>
```

Comment:
```suggestion
                "Federated": "arn:aws:iam::777777777777:oidc-provider/<oidc-provider-id>" <1>
            },
            "Action": "sts:AssumeRoleWithWebIdentity",
            "Condition": {
                "StringEquals": {
                    "<oidc-provider-id>:sub": "system:serviceaccount:aws-load-balancer-operator:aws-load-balancer-operator-controller-manager" <2>
```

Reply:
Thanks, I missed those. I also replaced one more instance in the same file.